autofocus/predict/app/routes/predict.py

```diff
@@ -0,0 +1,21 @@
+from flask import Blueprint, jsonify, request
+
+from ..filesystem.TemporaryFile import TemporaryFile
+from ..prediction.prediction import predict
+from ..validation.predict import validate_predict_request
+
+
+predict_route = Blueprint("predict", __name__)
```

Comment:
I've never used `Blueprint` objects, but I'm intrigued. Can you point me to a resource that explains how they work?